openhabian-setup.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2034
```

Comment:
Can this be avoided?

Reply:
yes but then either
 - we need it multiple times 
 - we export these variables which is what I did before. But Elias didn't like it.

Reply:
In this case I would say a global `shellcheck disable=SC2034` is preferred to not clutter the code.